core/unify.ml

```diff
@@ -1022,24 +1095,30 @@ and unify_rows' : ?var_sk:Subkind.t -> unify_env -> ((row * row) -> unit) =
     unify_both_rigid_with_rec_env rec_env (lrow, rrow) in
 
   let unify_one_rigid ((rigid_field_env, _, _ as rigid_row), (flexible_field_env, _, _ as flexible_row)) =
-    let (rigid_field_env', rigid_row_var', rigid_dual') as rigid_row', rigid_rec_row = unwrap_row rigid_row in
-    let closed = is_closed_row rigid_row' in
-    let (flexible_field_env', flexible_row_var', flexible_dual) as flexible_row', flexible_rec_row = unwrap_row flexible_row in
+    (* FIXME: row = typ was an idiotic mistake! *)
```

Comment:
Issue?

Reply:
This will be dealt with in phase 2 of type refactoring - it relates to the `row'` / `row_info` question